iamproperty/iamproperty.github.io
# Ds 46/search in nav

## Summary of Changes:
1. Update the CSS
2. Update examples

## Rationale
Update the nav component to handle search functionality.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/nav then review the 'Add search form' section

## Ticket(s)
DS-46

## Notes
